### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by t-bello7**
- Create Rails application with Postgresql database
- Create Rails API for randomly generating greetings
- Add Linter configurations

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Linters
+
+on: pull_request
```
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:3]: - [ ] In your Frontend repository readme file, you should have a link to your Backend repository :+1: to follow the requirements :blush:

Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:3]: In the pull request summary of Frontend repository, you should have a link to your Backend pull request :+1: to follow the requirements :blush:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

## Hey @t-bello7 :wave:

## Highlights :1st_place_medal:

✔️ Meaningful Commit Messages
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @t-bello7,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**